how-we-work/mozdotorgdurable.md

```diff
@@ -1,74 +1,123 @@
+
+```
+└── Project Board
+    ├── Moz.org Durable Team Board
+    ├── Project #1 Board
+    ├── Project #2 Board
+    ├── Project #3 Board
+    └── Project #4 Board
+```
+
+### Project Board
+
+Significant projects are tracked on our [Project Board](https://github.com/mozilla/bedrock/projects/3).
+
+Rules for this board are as follows:
+
+1. Any significant project (defined as work containing multiple tasks with multiple people) must have a card on the board.
+2. The project on the board must have a description containing links to any background information and that projects task board. 
+3. Keep the contents of these cards limited, discussiom about actual work should occur within those tasks.
+4. Only the following labels can be applied to projects:
+      1. The `Project` label.
+      2. The correct `Team` label. One of the following: Retention, Consumers, Firefox Growth, or Moz.org . 
+      3. Blocked if blocked.
+5. No other labels should be applied to projects.
+6. Folks who are working on a project, should be assigned.
+
+#### Project Status Board Columns
+
+- `Icebox` - Projects that have been prioritized to occur next.
+- `Ready` - Prioritized projects that are ready to start.
+- `In Progress` - Projects that are currently underway.
+- `Done` - Shipped!
+
+
+### Moz.org Durable Team Board
+
+This board is for tasks and projects initiated by the Moz.org durable team. It operates by the same set of rules and
+best practices as our individual project boards. See below.
+
+
+### Project Boards
+
+Any significant project should have it's own project board where work is managed. When work on a project is complete, the board can be closed.
 
 ## Board Details
 
-* What are your columns, what do the columns mean?
+### What are your columns, what do the columns mean?
+
+Generally our projects will have the following columns:
+
+- `Backlog` - Tasks we know have to get done but are not necessarily ready to go.
+- `Ready` - Prioritized tasks that are ready to start.
+- `In Progress` - Tasks that are currently underway.
+- `Done` - Shipped!
 
-Board columns:  Incoming | Research | Ready | In progress | Review | Done | Archived 
+### How/when do cards move across the board?
 
-* How/when to cards move across the board?
+Cards are moved by any member of the team when a change in status is identified. Cards should generally be moved by whomever takes the work or causes an update to status. This is not the responsibility of the Program Manger, team members show own the cards they take on including keeping the status up to date.
```

Comment:
- one typo "Program Manager"

- (Request to add second and third paragraphs)
+Moving a card represents a critical juncture in a project where the potential for delay and/or misunderstanding during transition is high. It's important that these actions be communicated (and acknowledged), especially if there's *any* lack of clarity on what needs to happen next. 

Further, cards can 'move' within a column, meaning that the ownership (the person doing the bulk of the work at that time) of a card can transfer. While a card is In Progress on a project it will transition, necessarily, though the team / team subset will still be providing input, to a state where one person is the principal owner.